integration_tests/tests/sql/test_get_filtered_columns_in_relation.sql

```diff
@@ -0,0 +1,5 @@
+{% set actual_output = dbt_utils.get_filtered_columns_in_relation(from= ref('data_star_uppercase_columns'), except=['field_1'], output_lower=True) %}
+
+{% set expected_output = ['field_2', 'field_3'] %}
+
+{{ assert_equal_values (actual_output | trim, expected_output | trim) }}
```

Comment:
This is cool 🔥  out of curiosity, is object equivalency in Jinja/Python order-dependent? i.e. would `['field_3', 'field_2']` pass? I don't have strong feelings about whether it should or not (if you pushed me on it, I'd say they're _not_ equal), but feels important to know.

Reply:
They are not equal. If you did want them to be treated that way, you would need to do something like: 
```
sorted(expected_output) == sorted(actual_output)
```
![Screen Shot 2022-03-26 at 9 21 09 AM](https://user-images.githubusercontent.com/30663534/160209806-3915c701-be0e-471d-80b7-4c6cfca7b3fe.png)

Reply:
TIL! thank you 🙏